ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -224,6 +218,15 @@ const ProfileMainScreen = ({ hideModal }: Props) => {
   );
 
   const logoutLabel = I18n.t("profile.logout.menulabel");
+  useHeaderSecondLevel({
+    title: "",
+    supportRequest: true,
+    faqCategories: ["profile"],
+    contextualHelpMarkdown: {
+      title: "profile.main.contextualHelpTitle",
+      body: "profile.main.contextualHelpContent"
+    }
+  });
```

Comment:
We'd put this under our FF `isSettingsVisibleAndHideProfile`. Am I right? I see the Header 2^ Level instead of the first navigating from `TabBar`.

<img src="https://github.com/pagopa/io-app/assets/16268789/a147ce30-d3b3-4662-9651-1994a81e3a13" width="250" />

Reply:
yes! Good catch!

Reply:
Thanks!

Reply:
done! Unfortunatelly, I couldn't use `useHeaderSecondLevel` because it cannot be render conditionally, so I used useLayoutEffect